src/rules/flake8_bandit/rules/try_except_pass.rs

```diff
@@ -0,0 +1,45 @@
+use ruff_macros::derive_message_formats;
+use rustpython_ast::{Expr, ExprKind, Located, Stmt, StmtKind};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::define_violation;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct TryExceptPass;
+);
+impl Violation for TryExceptPass {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Try, Except, Pass detected.")
+    }
+}
+
+/// S110
+pub fn try_except_pass(
+    checker: &mut Checker,
+    type_: Option<&Expr>,
+    _name: Option<&str>,
+    body: &[Stmt],
+    check_typed_exception: bool,
+) {
+    if body.len() == 1
+        && body[0].node == StmtKind::Pass
+        && (check_typed_exception
+            || match &type_ {
```

Comment:
We can do a bit better here. Try: `checker.resolve_call_path(&type).map_or(false, |call_path| call_path.as_slice() == ["", "Exception"])`.

That will verify that `&type` is a reference to the built-in `Exception` (so it also does the right thing if `Exception` gets shadowed and assigned to some other variable -- which is weird, but correct).

Reply:
Fixed in my latest commit, although I didn't add a test for the case of `Exception` being shadowed.